command/loadgen_test.go

```diff
@@ -0,0 +1,110 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/filecoin-project/storetheindex/command/loadgen"
+	"github.com/filecoin-project/storetheindex/config"
+	"github.com/stretchr/testify/require"
+	"github.com/urfave/cli/v2"
+)
+
+const FinderAddr = "/ip4/127.0.0.1/tcp/13000"
+const IngestAddr = "/ip4/127.0.0.1/tcp/13001"
+const AdminAddr = "/ip4/127.0.0.1/tcp/13002"
+
+func TestSmallLoad(t *testing.T) {
+	ctx, cncl := context.WithTimeout(context.Background(), 60*time.Second)
+	defer cncl()
+	testLoadHelper(ctx, t, 1, 1000, false)
+}
+
+func TestSmallLoadOverHTTP(t *testing.T) {
+	ctx, cncl := context.WithTimeout(context.Background(), 60*time.Second)
+	defer cncl()
+	testLoadHelper(ctx, t, 1, 1000, true)
+}
+
+func TestLargeLoad(t *testing.T) {
+	ctx, cncl := context.WithTimeout(context.Background(), 180*time.Second)
```

Comment:
I may skip this in our current CI, but at least having a large load test in CI is pretty straightforward.

Reply:
ah this finishes in 15s on the mac runner and 7s on the ubuntu runner. I'll keep it since it's our only concurrent integration test.